spec/oncocore_vs.txt

```diff
@@ -1,21 +1,16 @@
 Grammar:	ValueSet 5.0
-Namespace:	oncocore
+Namespace:	onco.core
 
 CodeSystem:     SCT = http://snomed.info/sct
 CodeSystem:	    MTH = http://ncimeta.nci.nih.gov
 CodeSystem:     ICD10CM = http://hl7.org/fhir/sid/icd-10-cm
-CodeSystem:     ICDO3MORPH = http://codes.iarc.fr/codegroup/2
-CodeSystem:     ICDO3TOPO = http://codes.iarc.fr/topography 
-
 
 ValueSet:     CancerBodyLocationVS
-Description:  "Codes describing the location of primary or secondary cancer. The value set includes all codes from ICD-O-3 topology and SNOMED-CT body locations."
-Includes codes from ICDO3TOPO
+Description:  "Codes describing the location of primary or secondary cancer. The value set includes all codes SNOMED-CT body structure hierarchy. The cancer body location code can also come from ICD-O-3 topography codes, however, those codes are not included here due to intellectual property restrictions. Because the value set binding is extensible in the information model, use of ICD-O-3 topography codes conform to mCODE requirements."
 Includes codes descending from SCT#123037004  "Body Structure"
 
 ValueSet:   HistologyMorphologyBehaviorVS
-Description: "The histologic type codes can be taken either from ICD-O-3 morphology codes (including behavior suffix) or SNOMED CT neoplasm morphology codes. For primary cancers, the ICD-O-3 behavior suffix should be /1, /2, or /3. For secondary cancers, the ICD-O-3 code should be /6 or /9."
-Includes codes from ICDO3MORPH
+Description: "The histologic type codes can be taken either from ICD-O-3 morphology codes (including behavior suffix) or SNOMED CT neoplasm morphology codes. For primary cancers, the ICD-O-3 behavior suffix should be /1, /2, or /3. For secondary cancers, the ICD-O-3 behavior suffix should be /6 or /9. The ICD-O-3 codes are not included here due to intellectual property restrictions, but the value set binding is extensible, so ICD-O-3 codes conform to mCODE requirements."
```

Comment:
Would not include notes on information model in value set description.

Reply:
I actually read this as a pre-coordinated term in the value set so I would keep this description here as-is.